python/Ganga/GPIDev/Lib/GangaList/GangaList.py

```diff
@@ -421,7 +434,10 @@ def __str__(self):
         return self.toString()
 
     def append(self, obj, my_filter=True):
-        self._list.append(self.strip_proxy(obj, my_filter))
+        elem = self.strip_proxy(obj, my_filter)
+        if isType(elem, GangaObject):
+            stripProxy(elem)._setParent(stripProxy(self)._getParent())
```

Comment:
Also, just so I understand what's happening here; when an item is added to a `GangaList` its parent is set to be same as the parent of the `GangaList`? So in the case of a `Job` with subjobs, any subjobs have their parent set to be the job and not the `GangaList` itself? I think that makes sense, I just wanted to check.